ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -339,7 +339,8 @@ private EndpointResponse handleQuery(final KsqlSecurityContext securityContext,
 
     if (statement.getStatement().isPullQuery()) {
       final ImmutableAnalysis analysis = ksqlEngine
-          .analyzeQueryWithNoOutputTopic(statement.getStatement(), statement.getStatementText());
+          .analyzeQueryWithNoOutputTopic(
+              statement.getStatement(), statement.getStatementText(), request.getConfigOverrides());
```

Comment:
nit: use `configProperties` instead of `request.getConfigOverrides()` (and also below). The way that the methods in this class are set up is really confusing and would probably benefit from a larger refactor but that's outside the scope of this fix.